src/utils.js

```diff
@@ -50,3 +50,7 @@ export const burnItDown = (message?: BurnItDownResponse) => {
     unsubscribe();
   }, timeToBurnItDown);
 };
+
+export const sortByTimestamp = (messages: Array<{timestamp: number}>): Array<Object> => {
+  return messages.sort((a, b) => a.timestamp - b.timestamp);
```

Comment:
This mutates the original array. This method should return a copy of the array that's sorted